src/main/java/com/dnsimple/endpoints/Domains.java

```diff
@@ -268,17 +268,17 @@ public PaginatedResponse<EmailForward> listEmailForwards(Number account, String
     /**
      * Create an email forward for a domain.
      *
-     * @param account The account ID
-     * @param domain  The domain name or ID
-     * @param from    The email address the emails are send to
-     * @param to      The email address the email address the emails are forwarded to.
+     * @param account          The account ID
+     * @param domain           The domain name or ID
+     * @param aliasEmail       The email address the emails are send to
+     * @param destinationEmail The email address the email address the emails are forwarded to.
      * @return The create email forward response
      * @see <a href="https://developer.dnsimple.com/v2/domains/email-forwards/#createEmailForward">https://developer.dnsimple.com/v2/domains/email-forwards/#createEmailForward</a>
      */
-    public SimpleResponse<EmailForward> createEmailForward(Number account, String domain, String from, String to) {
+    public SimpleResponse<EmailForward> createEmailForward(Number account, String domain, String aliasEmail, String destinationEmail) {
         Map<String, String> options = new HashMap<>();
-        options.put("from", from);
-        options.put("to", to);
+        options.put("alias_email", aliasEmail);
```

Comment:
This should be `alias_name`. `alias_email` is the name of the attribute on the read model.